src/i18n/locales/en/translation.json

```diff
@@ -1116,7 +1107,7 @@
     "error": "Error",
     "general": "General",
     "settings": "Extension Settings",
-    "accounts": "Wallets",
+    "wallets": "Wallets",
```

Comment:
You changed the key here?

Reply:
aah it seems its not used anywhere. removed from all the translation files

Reply:
thinking to create flow to detect and remove unused keys for translations in separate pr

Reply:
you can't because they are dynamically generated at some places, please don't invest any time and double check that this key really wasn't used :+1:

Reply:
yeah i double checked with search bar and if we remove key that is in used "yarn tsc:compile" will fail so yeah this is not being used anywhere

Reply:
> "yarn tsc:compile" will fail so yeah

not if keys are dynamically generated (which I don't think is the case here)

Reply:
treid git blame as well. it was used in userMenu before. we don't put accounts in userMenu anymore. so we can safely remove this translations

https://github.com/getAlby/lightning-browser-extension/commit/141fef9de50963168359b74ef906bc004b679633#diff-18393fdc0e154ed1578d0a767e77e6abf3ccd7debbbf2fd7266bc568e5f937acR99